### DISCUSSION TITLE
Is Splade Multilingual? If not how to

### DISCUSSION THREAD

**Discussion by wilfoderek** [castorini/pyserini#1487] [Q&A]
Hi there!
I was wonder if this tool can be multilingual if not how should i try to make it happen. 
Whre can i find practical examples of this tool.
Thank you in advance.

**Comment by cadurosar** [comment]
Hi Wilfo,

There's a version of SPLADE which is multilingual, but it is still pretty preliminary and the models are not available online. A not-so-polished description is available here https://arxiv.org/pdf/2302.14723.pdf but we still need to work more on that.
